src/test/java/PersonalAccountParameterizedTest.java

```diff
@@ -0,0 +1,105 @@
+    private final String browser;
+    private final String password;
+    private final String email;
+    private final String name;
+    private final By profileButton = By.xpath(".//a[text()='Профиль']");
+    public PersonalAccountParameterizedTest(String name, String email, String password, String browser){
+        this.name = name;
+        this.email = email;
+        this.password = password;
+        this.browser = browser;
+    }
+
+    @Parameterized.Parameters(name = "Проверка логина и логаута из личного кабинета пользователя: {1}, браузер: {3}")
+    public static Object[][] checkPersonalAccount() {
+        return new Object[][]{
+                {"Sugar-2", "yes_please-02@mail.ru", "123456S", "chrome"},
+                {"Solt-2", "cry_cry_baby-02@mail.ru", "123456S", "yandex"}
+        };
+    }
+
+    @Before
+    public void setup() {
+        driver = SetUpBrowser.getWebDriver(browser);
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.get(SetUpBrowser.getBaseUrl());
+        accessToken = null;
+        userRequest = new BaseUserSteps();
+    }
+
+    @Test
+    @DisplayName("Переход в Личный кабинет по кнопке Личный кабинет с главной страницы")
+    public void checkEnterPersonalAccountFromMainPage() {
+        UserRequest userRequestForDelete;
+
+        CreateBaseUser.createBaseUser(name,email,password,driver);//создали пользователя
+        LoginBaseUser.logInBaseUser(password,email,driver);//залогинили пользователя
+
+        userRequestForDelete = getUserRequest(name, email, password);
+        ValidatableResponse loginResponse = userRequest.loginUser(userRequestForDelete);
+        loginResponse.assertThat().statusCode(SC_OK).and().body("success", equalTo(true));
+        accessToken = loginResponse.extract().path("accessToken");//получили токен пользователя
+
+        MainPage mainPage = new MainPage(driver);
+        mainPage.enterPersonalAccount();
+        String expectedResult = "В этом разделе вы можете изменить свои персональные данные";
+        MatcherAssert.assertThat("Ошибка регистрации",driver.findElement(profileButton), anything(expectedResult));
+    }
+
+    @Test
+    @DisplayName("Выход из Личного кабинета")
+    public void checkLogOutFromPersonalAccount() {
+        UserRequest userRequestForDelete;
+
+        CreateBaseUser.createBaseUser(name,email,password,driver);//создали пользователя
+        LoginBaseUser.logInBaseUser(password,email,driver); //залогинили пользователя
+
+        userRequestForDelete = getUserRequest(name, email, password);
+        ValidatableResponse loginResponse = userRequest.loginUser(userRequestForDelete);
+        loginResponse.assertThat().statusCode(SC_OK).and().body("success", equalTo(true));
+        accessToken = loginResponse.extract().path("accessToken");//получили токен пользователя
+
+        MainPage mainPage = new MainPage(driver);
+        mainPage.enterPersonalAccount();
+        LoginBaseUser.logOutBaseUser(driver);
```

Comment:
Ошибка:
org.openqa.selenium.ElementClickInterceptedException: Element <p class="AppHeader_header__linkText__3q_va ml-2"> is not clickable at point (1192,44) because another element <div class="Modal_modal_overlay__x2ZCr"> obscures it

Reply:
Заработало с обновлением хрома и драйвера. Это замечание неактуально.